Dockerfile

```diff
@@ -24,7 +24,8 @@ COPY . .
 
 RUN go test -v ./...
 ENV CGO_ENABLED=0
-RUN go install -v ./...
+RUN BUILD_VERSION=$(git describe --tags --always --dirty --match=v* || (echo "command failed $$?"; exit 1)) \
+ && go install -v -ldflags "-X main.Version=$BUILD_VERSION -X 'main.BuildDate=$(date)'" ./...
```

Comment:
Why not pass the version from GH action with Docker's ARG instruction?

Dockerfile:
```
ARG BUILD_VERSION=unknown
```
`docker build --build-arg BUILD_VERSION=${VERSION} ...`